api_app/api/routes/airlock.py

```diff
@@ -41,6 +41,22 @@ async def create_draft_request(airlock_request_input: AirlockRequestInCreate, us
     return AirlockRequestInResponse(airlockRequest=airlock_request)
 
 
+@airlock_workspace_router.get("/workspaces/{workspace_id}/requests",
+                              status_code=status.HTTP_200_OK,
+                              response_model=AirlockRequestInList,
+                              name=strings.API_LIST_AIRLOCK_REQUESTS,
+                              dependencies=[Depends(get_current_workspace_owner_user), Depends(get_workspace_by_id_from_path)])
```

Comment:
why is this only available for ws owner?
```suggestion
                              dependencies=[Depends(get_current_workspace_owner_or_researcher_user), Depends(get_workspace_by_id_from_path)])
```